connectors/email/src/test/java/io/camunda/connector/email/integration/BaseEmailTest.java

```diff
@@ -214,8 +214,7 @@ protected List<String> getTo(Message message) {
   protected String getLastMessageId() {
     Message message = getLastReceivedEmails()[0];
     try {
-      String messageId = message.getHeader("Message-ID")[0];
-      return messageId.trim().replaceAll("[<>]", "");
+      return message.getHeader("Message-ID")[0];
```

Comment:
This method could use: https://github.com/camunda/connectors/pull/4742/files#diff-0cdc6a9ff56610a7c6af5e92b5942f5bef23354885215b037ae4bd82247fade6R313